src/main/kotlin/org/openrewrite/RecipeMarkdownGenerator.kt

```diff
@@ -886,7 +885,7 @@ class RecipeMarkdownGenerator : Runnable {
                         {% tab title="Maven Command Line" %}
                         {% code title="shell" %}
                         ```shell
-                        mvn -U org.openrewrite.maven:rewrite-maven-plugin:run \
```

Comment:
Not all maven projects use the maven wrapper. Some people may be confused by the use of mvnw

Reply:
👍 Good to know. I was testing this with our spring petclinic repo and I didn't have `mvn` installed so I only was able to run with the wrapper. I'll update the doc to remove the wrapper but at a note saying you will need to have Maven installed.